### ISSUE TITLE
How to decrease the detected edge size?

### ISSUE THREAD

**Issue by shubham12tomar** [xavysp/DexiNed#74]
Dear Sir,

1. I need thin edges but the predicted edges are thick. Can you please tell me the parameter so that I can make edges thin the in predicted outputs.

Thanks and Regards.

**Comment by xavysp** [comment]
Hi, its a great question and  the edge thinness  depends on different things. For example if you compare Dexined trained in BIPED   en evaluated on BSDS; and Dexined trained on BSDS tested on BSDS, the  predicted edge maps from the first one are thinner.
But if you want more thinner, it depends on loss function, GT, architecture, parameters tuning.

Another option is apply NMS to the predicted edge-maps.

 Hope it helped

**Comment by weinixuehao** [comment]
What is (GT, NMS)?

**Comment by g-h-anna** [comment]
> What is (GT, NMS)?

GT = ground truth, NMS = non-maximum suppression

@xavysp What do you mean by parameter tuning? If I use the same architecture (no changes in model.py) the same GT, same loss, as in a training before, than how can it happen now when I runned it again, that I have thicker edges learned? What have I might changed? Should I modify the fusion's weights?

**Comment by neo133** [comment]
any update on this? I am also facing the same issue with thick edges. Could you please guide as to how to apply NMS on the edge map? @xavysp

**Comment by xavysp** [comment]
> any update on this? I am also facing the same issue with thick edges. Could you please guide as to how to apply NMS on the edge map? @xavysp

HI sorry for this answare, please go to RFC repo, the official one, the you may find the code and a more pedagogical explanation
